packages/flex/package.json

```diff
@@ -14,35 +14,36 @@
     "cleanup": "rimraf node_modules .svelte-kit dist"
   },
   "devDependencies": {
-    "@sveltejs/adapter-auto": "^2.0.0",
-    "@sveltejs/kit": "^1.20.4",
-    "@sveltejs/package": "^2.1.0",
+    "@sveltejs/adapter-auto": "^3.1.1",
+    "@sveltejs/kit": "^2.4.3",
+    "@sveltejs/package": "^2.2.6",
+    "@sveltejs/vite-plugin-svelte": "^3.0.0",
     "@threlte/core": "workspace:*",
     "@threlte/extras": "workspace:*",
-    "@types/node": "^18.0.3",
-    "@types/three": "^0.158.3",
-    "@typescript-eslint/eslint-plugin": "^5.45.0",
-    "@typescript-eslint/parser": "^5.45.0",
-    "@yushijinhun/three-minifier-rollup": "^0.3.1",
-    "eslint": "^8.28.0",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-svelte": "^2.30.0",
-    "prettier": "^2.8.8",
-    "prettier-plugin-svelte": "^2.10.1",
-    "publint": "^0.1.12",
-    "rimraf": "^5.0.1",
-    "svelte": "^4.1.1",
-    "svelte-check": "^3.4.3",
-    "svelte-preprocess": "^5.0.4",
-    "svelte2tsx": "^0.6.19",
-    "three": "^0.158.0",
-    "tslib": "^2.4.1",
-    "typescript": "^5.0.0",
-    "vite": "^4.3.6"
+    "@types/node": "^20.11.6",
+    "@types/three": "^0.160.0",
+    "@typescript-eslint/eslint-plugin": "^6.19.1",
+    "@typescript-eslint/parser": "^6.19.1",
+    "@yushijinhun/three-minifier-rollup": "^0.4.0",
+    "eslint": "^8.56.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-svelte": "^2.35.1",
+    "prettier": "^3.2.4",
+    "prettier-plugin-svelte": "^3.1.2",
+    "publint": "^0.2.7",
+    "rimraf": "^5.0.5",
+    "svelte": "^4.2.9",
+    "svelte-check": "^3.6.3",
+    "svelte-preprocess": "^5.1.3",
+    "svelte2tsx": "^0.7.0",
+    "three": "^0.160.1",
+    "tslib": "^2.6.2",
+    "typescript": "^5.3.3",
+    "vite": "^5.0.12"
   },
   "dependencies": {
     "mitt": "^3.0.1",
-    "yoga-layout": "^2.0.0"
+    "yoga-layout": "^2.0.1"
```

Comment:
There's one minor here but I'm guessing not many, if any, would be worried about it

Reply:
Ah missed that one! I'll add a changeset to flex 👍